ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFDateFormat.java

```diff
@@ -111,17 +123,18 @@ public Object evaluate(DeferredObject[] arguments) throws HiveException {
     // the function should support both short date and full timestamp format
     // time part of the timestamp should not be skipped
     Timestamp ts = getTimestampValue(arguments, 0, tsConverters);
+
     if (ts == null) {
       Date d = getDateValue(arguments, 0, dtInputTypes, dtConverters);
```

Comment:
You can also remove this line because it is already taken care as part of timestamp.

Reply:
Done

Reply:
Also remove unused methods - 

Date.toEpochMilli(ZoneId id)
timestamp.toEpochMilli(ZoneId id)
timestamp.ofEpochMilli(long epochMilli, ZoneId id)